cylc/flow/network/schema.py

```diff
@@ -116,22 +126,6 @@ def sort_elements(elements, args):
     'Node': PROXY_NODES,
 }
 
-CYCLING_TYPES = [
-    'family_proxies',
-    'family_proxy',
-    'jobs',
-    'job',
-    'task_proxies',
-    'task_proxy',
-]
-
-PROXY_TYPES = [
-    'family_proxies',
-    'family_proxy',
-    'task_proxies',
-    'task_proxy',
-]
-
```

Comment:
These were apparently unused

Reply:
Must have developed past them, where used in resolvers